src/index.ts

```diff
@@ -7,8 +7,9 @@ const prisma = new PrismaClient();
 
 const app = express();
 
-app.use(express.static(path.join(__dirname, "../public")));
+app.use(express.urlencoded({ extended: false }));
 app.use(express.json());
+app.use(express.static(path.join(__dirname, "../public")));
 
```

Comment:
Remember you have to add your authenticate middleware into your app! Otherwise it won't do anything
```suggestion
app.use(authenticateJWT)
```

Reply:
@dyazdani add JWT middleware here